Makefile

```
@@ -4,19 +4,21 @@
 # You can set these variables from the command line.
 PYTHON        = python3
 VENVDIR       = ./venv
-BUILDDIR      = _build
-SPHINXOPTS    = -W --keep-going
 SPHINXBUILD   = $(VENVDIR)/bin/sphinx-build
+SPHINXOPTS    = -W --keep-going
+BUILDDIR      = _build
 BUILDER       = html
-SPHINXLINT    = $(VENVDIR)/bin/sphinx-lint
+JOBS          = auto
 PAPER         =
+SPHINXLINT    = $(VENVDIR)/bin/sphinx-lint
 
 # Internal variables.
 PAPEROPT_a4     = -D latex_paper_size=a4
 PAPEROPT_letter = -D latex_paper_size=letter
 ALLSPHINXOPTS   = -b $(BUILDER) \
                   -d $(BUILDDIR)/doctrees \
-                  $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) \
+                  $(PAPEROPT_$(PAPER)) \
+                  -j $(JOBS) $(SPHINXOPTS) \
```

Shall we put each `-x SOME_OPTION` in its own line, and then the `PAPEROPT_xxx` and `SPHINXOPTS` lucky bags on their own lines too?

(`PAPEROPT_xxx` is a `-D` under the hood, but we can't see it here.)

```suggestion
                  -j $(JOBS) \
                  $(PAPEROPT_$(PAPER)) \
                  $(SPHINXOPTS) \
```